mod/filter/set/right/browse_metric_filter.rb

```diff
@@ -55,27 +55,20 @@ def sort_options
     }
   end
 
-  view :metric_type_formgroup, cache: :never do
-    metric_type_select
-  end
-
-  view :research_policy_formgroup, cache: :never do
-    research_policy_select
-  end
-
-  view :wikirate_topic_formgroup, cache: :never do
-    autocomplete_filter :wikirate_topic
-  end
```

Comment:
Is this correct that we want different types of input fields for these?

Reply:
Well, it certainly looks like the browse metric filter was _trying_ to override the default filter formgroups.  But these were using the wrong view names for that (they would need to start with _filter_.  I'm not actually done with this PR yet, but most of the requests I've been getting are to standardize the interface (for topics, research policy, etc).  If I go back to overriding, I'll document the rationale.